### PR TITLE
issue#226

### PR DESCRIPTION
**Pull request by jackzampolin**
Fixes https://github.com/influxdb/influxdb.com/issues/226
@gunnaraasen Changed the format to what we talked about yesterday.

### CODE REVIEW

File: content/docs/v0.9/concepts/insights_tradeoffs.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  * *Con:* Update functionality is significantly restricted
+1. The vast majority of writes are for data with very recent timestamps and the data is added in time ascending order.
+  * *Pro:* Adding data in time ascending order is significantly more performant
+  * *Con:* Writing points with random times or with time not in ascending order is significantly less performant
+1. Scale is critical. The database must be able to handle a *high* volume of reads and writes. 
+  * *Pro:* The database can handle a *high* volume of reads and writes
+  * *Con:* The InfluxDB development team was forced to make tradeoffs to increase performance
+1. Being able to write and query the data is more important than having a strongly consistent view.
+  * *Pro:* Writing and querying the database can be done by multiple clients and at high loads
+  * *Con:* Query returns may not return the most recent points if database is under heavy load
```
Comment (beckettsean) [inline content/docs/v0.9/concepts/insights_tradeoffs.md]: Perhaps "Query returns may not _include_ the most recent points..."

**Comment by beckettsean** [comment]
:+1: from me.

@gunnaraasen?

**Comment by jackzampolin** [comment]
@gunnaraasen You good with this?

**Comment by beckettsean** [comment]
@jackzampolin @gunnaraasen let's fix and merge or kill the PR, I prefer the former

**Comment by jackzampolin** [comment]
Closing in favor of new PR with same info.

**Comment by jackzampolin** [comment]
https://github.com/influxdb/influxdb.com/pull/458

**Pull request closed without merging**